debugAdapterProtocol.json

```diff
@@ -1458,12 +1458,20 @@
 				},
 				"name": {
 					"type": "string",
-					"description": "The name of the variable's child to obtain data breakpoint information for.\nIf `variablesReference` isn't specified, this can be an expression."
+					"description": "The name of the variable's child to obtain data breakpoint information for.\nIf `variablesReference` isn't specified, this can be an expression, or an address if `asAddress` is also true."
 				},
 				"frameId": {
 					"type": "integer",
 					"description": "When `name` is an expression, evaluate it in the scope of this stack frame. If not specified, the expression is evaluated in the global scope. When `variablesReference` is specified, this property has no effect."
 				},
+				"bytes": {
```

Comment:
I am assuming that `bytes` is supposed to be a count of bytes, but the description doesn't say that.

Reply:
I tweaked it a little more and simplified it slightly, let me know what you think

Reply:
LGTM